expressEngine/lib/responseHandler/responseHandler.js

```diff
@@ -0,0 +1,17 @@
+import { error as commonError, success } from '../../cms/common';
+
+class responseHandler {
+  success = ({ data, message }) => ({ data: data || null, message: message || success.defaultSuccessMsg });
+
+  error = ({ error, status }) => ({ error: error || commonError.undefinedError, status: status || 500 });
```

Comment:
Replace
`  error = ({ error, status }) => ({ error: error || commonError.undefinedError, status: status || 500 });`
with this
`  error = ({ error = commonError.undefinedError, status = 500 }) => ({ error, status });`
Reason: Instead of adding extra checks, we can use default values while destucturing the object.